tmva/tmva/src/MethodFisher.cxx

```diff
@@ -127,7 +127,7 @@ The corresponding numbers are printed on standard output.
 
```

Comment:
Why this change ?
All ClassImp(....) have ";" at the end. It is maybe not relevant, but I would avoid touching files for something that is not needed, it pollutes always history and adds noise

Reply:
The rationale was because doxygen does not like well Preprocessor Macros ending with a semicolon. I agree that it's not needed, however if doxygen throws 1000 warnings of this type, it's hard to find the real warnings among the full list. But I can change it back.

Reply:
Reverted.